shared/src/search/interactive/util.ts

```diff
@@ -41,9 +41,20 @@ export enum FilterType {
     index = 'index',
```

Comment:
```suggestion
    index = 'index',
    stable = 'stable',
```
The `stable` one is missing from this list, I've been meaning to add it :-) https://github.com/sourcegraph/sourcegraph/issues/10056